packages/liquidator/src/liquidationStrategy.js

```diff
@@ -66,12 +61,11 @@ module.exports = (
   assert(BN, "requires BN");
 
   // these variables are optional and are only checked if WDF is activated
-  whaleDefenseFundWei &&
-    assert(
-      defenseActivationPercent >= 0 && defenseActivationPercent <= 100,
-      "Requires defenseActivationPercent to bet set between 0 and 100"
-    );
-  whaleDefenseFundWei && assert(withdrawLiveness > 0, "requires withdrawLiveness");
+  const wdfActive = !isNaN(defenseActivationPercent) && defenseActivationPercent > 0;
```

Comment:
nit: why did you choose isNaN? If you're trying to verify the type, would it be clearer to check `typeof defenseActivationPercent === "number"`?

Reply:
Yeah that's reasonable its a more strict check